OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -133,7 +133,10 @@ public void Tick(Squad owner)
 
 			var e = FindDefenselessTarget(owner);
 			if (e == null)
+			{
+				Retreat(owner, false, true, true);
```

Comment:
```suggestion
				Retreat(owner, flee: false, rearm: true, repair: true);
```

makes it a bit more readable.

Reply:
good idea.